dvc/lock.py

```diff
@@ -73,7 +74,11 @@ def lock(self):
             except flufl.lock.TimeOutError:
                 raise LockError(
                     "cannot perform the cmd since DVC is busy and "
-                    "locked. Please retry the cmd later."
+                    "locked. You can delete .dvc/lock to fix it but first, "
+                    "please make sure there is no other ongoing DVC process "
+                    "by running: {blue}`ps xaf | grep dvc`{nc}.".format(
```

Comment:
fix it but first, -> fix it, but first
there is no other ongoing -> there's no ongoing

Same in lines 129-130